frontend-implicaction/src/app/admin/jobs/admin-jobs.component.html

```diff
@@ -11,7 +11,107 @@ <h5 class="panel-title">Toutes les offres d'emplois</h5>
         title="Ajouter une offre"
       >
       </i-feather>
+      <button
+        (click)="toggleArchiveSelectedJobs()"
+        class="btn"
+        data-bs-placement="top"
+        data-bs-toggle="tooltip"
+        title="Archiver"
+        [disabled]="!isArchiveEnabled"
+        type="button"
+      >
+        <i
+          [ngClass]="isArchive ? 'fa-box border-info text-info':'fa-box-open border-warning text-warning' "
```

Comment:
```suggestion
          [ngClass]="isArchive ? 'fa-box border-info text-info':'fa-box-open border-warning text-warning'"
```
un p'tit espace oublié à la fin de ta ligne 😉